cam1: 
  distance_sensing_node:
    ros__parameters:
      iou_thresh: 0.3
      prob_thresh: 0.5
      config: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/yolov3.cfg"
      weights: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/yolov3.weights"
      class_names: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/coco.names"
      device: "cuda"
      output: "/home/protac/ros/protac_ws/src/protac_perception/resource/outputs/test-yolo-cam1.mp4"
      show_fps: true
      verbose: true

cam2: 
  distance_sensing_node:
    ros__parameters:
      iou_thresh: 0.3
      prob_thresh: 0.5
      config: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/yolov3.cfg"
      weights: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/yolov3.weights"
      class_names: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/coco.names"
      device: "cuda"
      output: "/home/protac/ros/protac_ws/src/protac_perception/resource/outputs/test-yolo-cam2.mp4"
      show_fps: true
      verbose: true

cam3: 
  distance_sensing_node:
    ros__parameters:
      iou_thresh: 0.3
      prob_thresh: 0.5
      config: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/yolov3.cfg"
      weights: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/yolov3.weights"
      class_names: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/coco.names"
      device: "cuda"
      output: "/home/protac/ros/protac_ws/src/protac_perception/resource/outputs/test-yolo-cam3.mp4"
      show_fps: true
      verbose: true

cam4: 
  distance_sensing_node:
    ros__parameters:
      iou_thresh: 0.3
      prob_thresh: 0.5
      config: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/yolov3.cfg"
      weights: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/yolov3.weights"
      class_names: "/home/protac/ros/protac_ws/src/protac_perception/resource/models/coco.names"
      device: "cuda"
      output: "/home/protac/ros/protac_ws/src/protac_perception/resource/outputs/test-yolo-cam4.mp4"
      show_fps: true
      verbose: true